prow/config.yaml

```diff
@@ -616,7 +616,11 @@ deck:
     header_color: "#0b74de" # Kyma color
     logo: "/static/extensions/logo.svg"
     favicon: "/static/extensions/favicon.png"
-
+  rerun_auth_configs:
+    'test-infra':
```

Comment:
I think key should be 'kyma-project/test-infra'. See this comment https://github.com/kubernetes/test-infra/blob/7ab5bdec015fd06031bb89d51df076dd98499f74/prow/config/config.go#L655